classes/label.rb

```diff
@@ -0,0 +1,16 @@
+class Label < Item
```

Comment:
The `label` class has a `one to many` relationship with the `Item` class instead of inheriting from it as shown in the image blow:
![Screenshot 2023-08-08 122551](https://github.com/Mahamadou-Nouridine/catalog-of-my-things/assets/98597449/56b380bd-5542-412a-acfb-8c8020cb2f78)

Kindly, remove the relashionship and implement only the relation according to the following requirement:

- All Label class properties visible in the diagram should be defined and set up in the constructor method.
- add_item method in the Label class
        - should take an instance of the Item class as an input
        - should add the input item to the collection of items
        - should add self as a property of the item object (by using the correct setter from the item object)